reference_frame: base_link
links:
- link: gripper_link
  velocity_limit: 0.6
  symmetry: xz
  gripper_topic: /blabla_gripper
- link: head_camera_link
  velocity_limit: 0.2

